cyclonedx-bom/src/utilities.rs

```diff
@@ -1,14 +1,58 @@
+use crate::errors::BomError;
+use std::convert::TryFrom;
+
 /// Convert an optional list of a type
 ///
 /// Used to translate between a common structure in the data model for going between the model version and the specification version
 pub(crate) fn convert_optional_vec<A, B: From<A>>(value: Option<Vec<A>>) -> Option<Vec<B>> {
     value.map(convert_vec)
 }
 
+/*
+pub(crate) fn try_convert_optional_vec<A, B: TryFrom<A, Error = BomError>>(value: Option<Vec<A>>) -> Option<Result<Vec<B>, BomError>> {
+    value.map(try_convert_vec)
+}
+*/
```

Comment:
Commented out code, here and further down in this file. If it is not used, it should be removed - storing older code is what git is for.